bundles/org.openhab.binding.hue/src/main/java/org/openhab/binding/hue/internal/handler/Clip2ThingHandler.java

```diff
@@ -945,6 +946,7 @@ private boolean updateChannels(Resource resource) {
             case LIGHT:
                 if (fullUpdate) {
                     updateEffectChannel(resource);
+                    updateColorTemperatureAbsoluteChannel(resource);
```

Comment:
This will be called each time.
No way to do it only at init time ?

Reply:
It is only called `if (fullUpdate) {` .. which basically means only at init time. Although full updates are occasionally polled I think every 60 minutes during normal operation just in case..

Reply:
Ok.
In addition, after the first time, the old and new objects will be identical and so you will not post a changed event. Correct ?